Game2048/Game/Models/Actions/MoveLeft.cs

```diff
@@ -0,0 +1,12 @@
+﻿using AbstractActionClass;
+
+namespace ActionClass
```

Comment:
For namespaces usage, consider this option in case you want to have a namespace for abstract classes and another one for specific classes
```
namespace Actions { // asbtract classes }
namespace Actions.Movements { // concrete classes }
```